Phonebook.Frontend/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -24,10 +24,14 @@ export class DashboardComponent implements OnInit {
   @Select(BookmarksState)
   public bookmarkedPersons$: Observable<Person[]>;
   public removedLastPersons: Person[] | null = null;
-  constructor(private store: Store, private cd: ChangeDetectorRef) {}
+  public drawerOpenByDefault: boolean = true;
```

Comment:
```suggestion
  public drawerOpen: boolean = true;
```
Das `true` sagt schon aus das es by default offen ist.